src/components/ListItem.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <template>
+      <v-container>
+        <v-row>
+          <v-col col="12" md="6">
+            <v-text-field
+              style="margin-bottom:20px;"
+              :value="listItem.text"
```

Comment:
I think using `v-model` should work instead of `:value` and `@change`.

Reply:
v-model was used and v-text-field was lost focus after one character. I know`:value `and `@ change` are as same as using` v-model ,` yet the proposed solution on StackOverflow was using `:value` and `@change`